src/main/java/com/hoverla/bring/context/AnnotationApplicationContextImpl.java

```diff
@@ -68,6 +65,13 @@ public <T> T getBean(Class<T> beanType) {
                 .orElseThrow(() -> new NoSuchBeanException(format(NO_SUCH_BEAN_EXCEPTION_BY_TYPE, beanType.getSimpleName())));
     }
 
+    private Object mapToProxy(Object bean, Class<?> beanType) {
+        for (ProxyConfigurator proxyConfigurator : proxyConfigurators) {
+            bean = proxyConfigurator.replaceWithProxyIfNeeded(bean, beanType);
```

Comment:
At this line, if we will have more than one proxy it saves the last one, but now for only caching it should work fine. Please add TODO so that we should handle this case.

Reply:
I guess it will wrap proxy under proxy. I will add one more proxy to test it